nix/t1/release/doc.typ

```diff
@@ -0,0 +1,71 @@
+#import "@preview/codly:0.2.0": *
+#show: codly-init.with()
+#codly(languages: (
+  bash: (name: "Bash", icon: none, color: rgb("#CE412B")),
+))
+
+= T1 Docker Image Guidance
+
+== Released IP configs
+
+#let table-json(config) = {
+  let name = config.name
+  let param = config.parameter
+  let floatSupport = if param.extensions.first() == "Zve32f" [ True ] else [ False ]
+  let VRFRamType = param.vrfRamType.split(".").last()
+  let VRF = [#param.vrfBankSize Bank, #VRFRamType]
+  let lsuBankCnt = param.lsuBankParameters.len()
+  let beatByteCnt = param.lsuBankParameters.first().beatbyte
+  table(
+    columns: 6,
+    [*Config Name*], [*DLEN*],      [*VLEN*],      [*Float support*], [*VRF*], [*LSU*],
+    [*#name*],       [#param.dLen], [#param.vLen], [#floatSupport],   [#VRF],  [#lsuBankCnt bank, #beatByteCnt beatbyte],
+  )
+}
+
+#table-json(json("./config.json"))
+
+== Address Range
+
+#table(
+  columns: 3,
+  [*Range*],  [*Usage*],                      [*Address Range*],
+  [0-1G],     [Scalar Bank],                  [0x20000000],
+  [1-3G],     [DDR Bank (512M/bank)],         [0x40000000],
+  [3G-3G+2M], [SRAM Bank (256K/bank 8Banks)], [0xc0000000]
+)
+
+Scalar core cannot access DDR/SRAM due to floorplan limitation, for all DDR/SRAM access to these banks, users need to load/store from/to corresponding memory banks.
+
+== How to use the Docker image
+
+```bash
+# Load the image into docker registry
+docker pull ghcr.io/chipsalliance/t1:latest
+# There are two ways to access the shell:
```

Comment:
Add the `docker -v` option to tell user how to bind mount local folder(this is useful to let them add their own source files.)